plugins/producer_plugin/test/test_read_only_trx.cpp

```diff
@@ -150,8 +150,20 @@ BOOST_AUTO_TEST_CASE(no_read_only_threads) {
    test_trxs_common(specific_args);
 }
 
-// test read-only trxs on separate threads (with --read-only-threads)
-BOOST_AUTO_TEST_CASE(with_read_only_threads) {
+// test read-only trxs on 1 threads (with --read-only-threads)
+BOOST_AUTO_TEST_CASE(with_16_read_only_threads) {
```

Comment:
test name does not match the number f threads.

Reply:
done